app/Http/Controllers/Bible/StreamController.php

```diff
@@ -222,11 +224,18 @@ private function generateMultipleMp3HLS($fileset_id, $file_id_location)
             return $this->setStatusCode(404)->replyWithError('No Audio fileset found for the provided params');
         }
 
-        $bible_files = BibleFile::where([
-            'hash_id' => $audio_fileset->hash_id,
-            'book_id' => $parts[0],
-            'chapter_start' => $parts[1]
-        ])->get();
+        if ($parts[1]) {
+            $bible_files = BibleFile::where([
+                'hash_id' => $audio_fileset->hash_id,
+                'book_id' => $parts[0],
+                'chapter_start' => $parts[1]
+            ])->get();
+        } else {
```

Comment:
It is related with the next example: https://digitalbibleplatform.postman.co/workspace/DBP4-workspace~0a283f7f-06d9-467d-97f0-fef92d18bd3d/request/12519377-ff2132f5-c332-4a89-8821-76beaf537629

When you run the above example it is returning the URL: `"http://APP_ROOT_PATH/api/bible/filesets/SVCWBTS2DA/MAT---/playlist.m3u8"` in the path parameter. So, if you use the above request it will return an empty m3u8 list. But, with this current change it will return all mp3 files for the book id and hash id given.